values.yaml

```diff
@@ -104,9 +113,11 @@ global:
     enabled: ${ENABLE_SMTP}
     user_name: ${SMTP_USER}
     password: 
-      secret: ${SMTP_SECRET_NAME}
+      secret: gitlab-smtp-secret
```

Comment:
is this correct ?

Reply:
yes, kubernetes secret name is prefixed . You are able to choose the gcp secret name you like.